### ISSUE TITLE
Compatibility with Mac OS Catalina

### ISSUE THREAD

**Issue by gbrennon** [pokebyte/FinalBurn-X#11]
Hey everyone,

Is this compatible with mac os catalina?

**Comment by yanzhou-ca** [comment]
Yes, but for now it's more like a prototype, you can't even set rom path or configure video/audio.

**Comment by 0xe1f** [comment]
Short answer: not really
Long answer: yes, but...

Because FBX uses low-level input monitoring, keyboard support is broken in Catalina. You can try enabling "Input monitoring" in Privacy and Security settings in System Preferences, which might fix keyboard support, or you can just use a gamepad or joystick.

@1900forever - ROM path customizability isn't an option intentionally. If you really want a custom path, you can symlink the ROM directory to where you usually store your ROMs.

I stopped working on this emulator a while ago and moved on to [FBNeo](https://github.com/fbn-mac/FBNeo/releases). It should be fairly on par with FBX - except for input customizability and joystick support.

**Comment by yanzhou-ca** [comment]
@0xe1f Thanks for the reply. I actually stared FBNeo mac version as well. No ROM path customisation would be inconvenient anyway, since many people have more than one hard drive. Usually people would not store roms on their system drive because of the limited ssd space. If there are only a few roms would be fine, but if we have full rom-sets it would be difficult to manage. If it's possible, could you please add this function in the preferences?
